src/drafty.js

```diff
@@ -1153,19 +1153,9 @@ Drafty.format = function(content, formatter, context) {
     context);
 }
 
-/**
- * Shorten Drafty document and strip all entity data leaving just inline styles and entity references.
- * The <code>context</code> may expose a function <code>getFormatter(style)</code>. If it's available
- * it will call it to obtain a <code>formatter</code> for a subtree of styles under the <code>style</code>.
- * @memberof Drafty
- * @static
- *
- * @param {Drafty|string} original - Drafty object to shorten.
- * @param {number} length - length in characters to shorten to.
- * @param {Object} context - context provided to formatter as <code>this</code>.
- * @returns new shortened Drafty object leaving the original intact.
- */
-Drafty.preview = function(original, length, context) {
+// Takes in an `original` document (Drafty or string) and transforms it
+// using formatter.
```

Comment:
Add a note that the result of the transformation is in `state`. Or maybe change the signature to return it.

Reply:
Done.